docs/sdks/server-sdks/dotnet/quickstart.mdx

```diff
@@ -0,0 +1,224 @@
+
+```csharp
+// Get assignment parameters
+string flagKey = "my-neat-feature";
+string subjectKey = GetCurrentUser() ?? "anonymous";
+var subjectAttributes = new Dictionary<string, object>
+{
+    ["country"] = user.Country,
+    ["device"] = user.Device
+};
+string defaultValue = "default-value";
+
+// Make an assignment
+var variation = eppoClient.GetStringAssignment(
+    flagKey,
+    subjectKey,
+    subjectAttributes,
+    defaultValue
+);
+```
+
+### Assignment Types
+
+The SDK provides different assignment functions based on the type of value you need:
+
+| Function | Return Type |
+|----------|-------------|
+| `GetStringAssignment()` | String |
+| `GetBooleanAssignment()` | Boolean |
+| `GetJSONAssignment()` | JSON object |
+| `GetIntegerAssignment()` | Integer |
+| `GetNumericAssignment()` | Float |
+
+### Using Assignments
+
+After receiving an assignment, your application should implement logic to modify the user experience accordingly:
+
+```csharp
+var variation = eppoClient.GetStringAssignment(
+    flagKey,
+    subjectKey,
+    subjectAttributes,
+    defaultValue
+);
+
+// Render different components based on assignment
+switch(variation) 
+{
+    case "landing-page-a":
+        return RenderLandingPageA();
+    case "landing-page-b":
+        return RenderLandingPageB();
+    default:
+        return RenderLandingPageC();
+}
+```
+
+## Experiments
+
+While feature flags are useful, they do not send you any information about how your users are interacting with the feature. Experiments provide a way to collect data about these interactions using whichever logging and data warehousing system you prefer.
+
+To log events through the SDK, you need to define a logger that implements the `IAssignmentLogger` interface:
+
+<Tabs>
+<TabItem value="console" label="Console">
+
+```csharp
+internal class AssignmentLogger : IAssignmentLogger
+{
+    public void LogAssignment(AssignmentLogData assignmentLogData)
+    {
+        Console.WriteLine(assignmentLogData);
+    }
+}
+```
+
+</TabItem>
+<TabItem value="segment" label="Segment">
+
+```csharp
+internal class SegmentLogger : IAssignmentLogger
+{
+    private readonly Analytics analytics;
+
+    public SegmentLogger(Analytics analytics)
+    {
+        this.analytics = analytics;
+    }
+
+    public void LogAssignment(AssignmentLogData assignmentLogData)
+    {
+        analytics.Track("Eppo Randomization Assignment", assignmentLogData);
+    }
+}
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+In a production application, you would want to replace the `Console.WriteLine()` statement with an actual logging system. We have documentation on how to set up logging with multiple popular data warehouses and logging systems in the [Assignment page](/sdks/server-sdks/dotnet/assignments/#logging-data-to-your-data-warehouse).
+:::
+
+## Contextual Bandits
+
+For bandits, you'll need both an assignment logger and a bandit logger:
+
+```csharp
```

Comment:
Segment Bandit Logger example here:
https://github.com/Eppo-exp/dot-net-server-sdk?tab=readme-ov-file#assignment-and-bandit-action-logger

Reply:
Updated.